DEV_GUIDE.md

```diff
@@ -32,6 +35,11 @@ export JAVA_HOME="/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/
 export PATH="$JAVA_HOME/bin:$PATH"
 ```
 
+### API Services
+The Stargate API services that run externally to the coordinator node are located under the `apis` directory. These services require a more modern JDK in order to take advantage of the latest tools and frameworks. See the [APIs README](apis/README.md) for information on compiling and running the API services including the required JDK. 
+
+(The remainder of these instructions focus on working with the Stargate coordinator node.)
+
```

Comment:
Need to delete some of next paragraph, especially `./mvnw package -pl restapi -am` is no longer possible (well it won't when we delete it, but until then it won't build new REST api).

Reply:
replacing `restapi` with`cql`